Cargo.toml

```diff
@@ -6,14 +6,15 @@ description = "rust libary for Lapce plugin"
 edition = "2021"
 
 [dependencies]
-anyhow = "1.0"
 bytes = "1"
 http = "0.2"
 wasi-experimental-http = { git = "https://github.com/lapce/wasi-experimental-http" }
 once_cell = "1.13.0"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0.59"
 jsonrpc-lite = "0.5.0"
+thiserror = "1.0.38"
+anyhow = "1.0.68"
```

Comment:
dependencies still point to specific versions instead of major ones

Reply:
Oh you meant that